pontoon/base/tests/forms/test_permission_log.py

```diff
@@ -1,77 +1,20 @@
-import factory
 import pytest
```

Comment:
While evaluating previous changesets, I already wrote a comment about the need of reusing fixtures. Now, I can delete it. :)